src/kontrol/foundry.py

```diff
@@ -550,7 +553,91 @@ def foundry_show(
             res_lines += print_failure_info(proof, kcfg_explore, counterexample_info)
             res_lines += Foundry.help_info()
 
-    return '\n'.join(res_lines)
+    if to_kevm_claims:
+
+        def _remove_foundry_config(_cterm: CTerm) -> CTerm:
+            kevm_config_pattern = KApply(
+                '<generatedTop>',
+                [
+                    KApply('<foundry>', [KVariable('KEVM_CELL'), KVariable('CHEATCODES_CELL')]),
+                    KVariable('GENERATEDCOUNTER_CELL'),
+                ],
+            )
+            kevm_config_match = kevm_config_pattern.match(_cterm.config)
+            if kevm_config_match is None:
+                _LOGGER.warning('Unable to match on <kevm> cell.')
+                return _cterm
+            return CTerm(kevm_config_match['KEVM_CELL'], _cterm.constraints)
+
+        def _contains_foundry_klabel(_kast: KInner) -> bool:
+            _contains = False
+            _foundry_labels = [
+                '#call_foundry___FOUNDRY-CHEAT-CODES_KItem_Int_Bytes',
+                '#error_foundry___FOUNDRY-CHEAT-CODES_KItem_Int_Bytes',
+                '#return_foundry___FOUNDRY-CHEAT-CODES_KItem_Int_Int',
+                'FOUNDRY_WHITELISTCALL_FOUNDRY-CHEAT-CODES_ExceptionalStatusCode',
+                'FOUNDRY_WHITELISTSTORAGE_FOUNDRY-CHEAT-CODES_ExceptionalStatusCode',
+                '#updateRevertOutput___FOUNDRY-CHEAT-CODES_KItem_Int_Int',
+                '#checkRevert_FOUNDRY-CHEAT-CODES_KItem',
+                '#clearExpectRevert_FOUNDRY-CHEAT-CODES_KItem',
```

Comment:
Why only this subset of labels?

Reply:
These are the ones that showed up in the examples I went through with, but I can probably just grab all the symbols from the cheat-codes module.

Reply:
Done.